src/server/views/gtag.ejs

```diff
@@ -0,0 +1,23 @@
+window.dataLayer = window.dataLayer || [];
```

Comment:
should this be renamed to `.js` / moved into a `js` folder?

Reply:
No - this is a template that will be interpolated with values held by the backend